workflow/signals.py

```diff
@@ -208,14 +204,24 @@ def check_seats_save_user_groups(sender, instance, **kwargs):
         elif count == 0 and ROLE_VIEW_ONLY in added_groups:
             used_seats -= 1
 
-            # Update the amount of used seats
         org.chargebee_used_seats = used_seats
         org.save()
 
-        # Validate the amount of available seats based on the subscription
-        user_addon = get_addon_by_id('user', sub_addons)
-        if user_addon:
-            available_seats = user_addon.quantity
+        # Load subscription data from ChargeBee
+        sub_id = org.chargebee_subscription_id
+        if not sub_id:
+            logger.info('The organization {} does not have a '
+                        'subscription'.format(tola_user.organization.name))
+            return
+
+        try:
+            result = Subscription.retrieve(sub_id)
+            subscription = result.subscription
+        except APIError as e:
+            logger.warn(e)
+        else:
+            # Validate the amount of available seats based on the subscription
+            available_seats = subscription.plan_quantity
             if available_seats < org.chargebee_used_seats:
```

Comment:
In my opinion it is more natural: `used_seats > available_seats`. But just a personal opinion, you don't need to change it.